herald/apps.py

```diff
@@ -13,4 +14,18 @@ class HeraldConfig(AppConfig):
     name = 'herald'
 
     def ready(self):
+        from .models import Notification
+        from herald import registry
+
         self.module.autodiscover()
+
+        try:
+            # add any new notifications to database.
+            for index, klass in enumerate(registry._registry):
+                Notification.objects.get_or_create(notification_class=klass.__name__)
```

Comment:
I wonder if this is really necessary to have a model like this? Can we not just have a `DisabledNotification` model that just stores the notification class name rather than a foreign key to a `Notification` model? Then we could make a helper function like `disable_notification(user, PasswordResetNotification)`.